### PR TITLE
adding id and link for the EverySecondPathLevelParameterRule rule (#535)

### LINKED ISSUE #535: Provide Id and Link for the "EverySecondPathLevelParameter" Rule

### PR DESCRIPTION
**Pull request by maxim-tschumak**
Closes #535.

### CODE REVIEW

**Comment by netme** [comment]
Great! Can I ask you to do 'git pull --rebase upstream master' before pushing next time?

**Comment by netme** [comment]
:+1:

**Comment by maxim-tschumak** [comment]
:+1:

File: server/src/main/java/de/zalando/zally/rule/EverySecondPathLevelParameterRule.kt [hunk 1]
```diff
@@ -8,12 +8,10 @@ import org.springframework.stereotype.Component
 @Component
 class EverySecondPathLevelParameterRule : SwaggerRule() {
     override val title = "Every Second Path Level To Be Parameter"
-    // TODO: Provide URL
-    override val url = ""
+    override val url = "/#143"
     override val violationType = ViolationType.MUST
     override val code = "M005"
```
Comment (maschleg) [inline server/src/main/java/de/zalando/zally/rule/EverySecondPathLevelParameterRule.kt]: why not also switching the code to `M143`?

Comment (tkrop) [reply server/src/main/java/de/zalando/zally/rule/EverySecondPathLevelParameterRule.kt]: Yes, why?

**Comment by tkrop** [comment]
👍

File: server/src/main/java/de/zalando/zally/rule/EverySecondPathLevelParameterRule.kt (hunk 1 above)
Comment (maxim-tschumak) [reply server/src/main/java/de/zalando/zally/rule/EverySecondPathLevelParameterRule.kt]: It's internal id which will be deprecated eventually (remember the concept I sent around?). And, the relationship between Guideline rules and Zally checks is `1-to-n` which will lead to internal ids which are not unique.

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by netme** [comment]
:+1:

**Pull request merged**